src/metabase/models/revision/diff.clj

```diff
@@ -83,7 +83,9 @@
     (deferred-tru "changed this from a model to a saved question")
 
     [:display _ _]
-    (deferred-tru "changed the display from {0} to {1}" (name v1) (name v2))
+    (if v1
```

Comment:
the bug was calling because `v1` is nil and we were calling `(name nil)`.

even thought `report_card.display` is a required field but in https://github.com/metabase/metabase/blob/5326187971bfe0901e078eb95902232640ba31fa/src/metabase/models/revision.clj#L141

we're getting the revision details between the last revision and `nil`